game_patch/multi/server.cpp

```diff
@@ -197,12 +243,83 @@ void load_additional_server_config(rf::Parser& parser)
     }
 }
 
+// memory addresses for weapon stay exemption indexes
+constexpr std::pair<bool WeaponStayExemptionConfig::*, uintptr_t> weapon_exemptions[] = {
+    {&WeaponStayExemptionConfig::rocket_launcher, 0x00872458},
```

Comment:
there's really no need to have all those offsets hardcoded here. Just let the people configure list of class names, then at load time translate them to indices using `weapon_lookup_type`